tests/netmap_test.go

```diff
@@ -56,13 +56,28 @@ func TestDeploySetConfig(t *testing.T) {
 		"config", container.AliasFeeKey)
 }
 
-func dummyNodeInfo(acc neotest.Signer) []byte {
+type testNodeInfo struct {
+	signer neotest.SingleSigner
+	pub    []byte
+	raw    []byte
+}
+
+func dummyNodeInfo(acc neotest.Signer) testNodeInfo {
 	ni := make([]byte, 66)
 	rand.Read(ni)
 
-	pub, _ := vm.ParseSignatureContract(acc.Script())
+	s := acc.(neotest.SingleSigner)
```

Comment:
Requires neo-go version bump?

Reply:
Actually no, we just forgot to fix this after update.

Reply:
For some reason tests still fail, though work on my local machine. Updated neo-go version.